nova/virt/vmwareapi/special_spawning.py

```diff
@@ -306,9 +307,17 @@ def free_host(self, context):
                             'maintenance or became disconnected.')
                 return FREE_HOST_STATE_ERROR
 
+        # filter the VMs on the host, so we don't look at the non-movable
+        # DRS-created and -owned VMs
+        vcls_identifier = (constants.VCLS_EXTENSION_KEY,
+                           constants.VCLS_EXTENSION_TYPE_AGENT)
+        non_vcls_vms = [
+            (u, state) for u, h, state, m in self._get_vms_on_host(host_ref)
```

Comment:
```suggestion
            (u, state) for u, _, state, m in self._get_vms_on_host(host_ref)
```
making it clear we don't use it.